README.md

```diff
@@ -93,13 +97,16 @@ name. That seems to be the fairest way to arrange this table.
 | SML | [smlnj](http://www.smlnj.org/) |
 | Swift | [swiftlint](https://swift.org/) |
 | Tex | [proselint](http://proselint.com/) |
-| Text^^ | [proselint](http://proselint.com/) |
+| Texinfo | [proselint](http://proselint.com/)|
+| Text^ | [proselint](http://proselint.com/) |
 | TypeScript | [tslint](https://github.com/palantir/tslint), typecheck |
 | Verilog | [iverilog](https://github.com/steveicarus/iverilog), [verilator](http://www.veripool.org/projects/verilator/wiki/Intro) |
 | Vim | [vint](https://github.com/Kuniwak/vint) |
+| Vim help^ | [proselint](http://proselint.com/)|
+| XHTML | [proselint](http://proselint.com/)|
 | YAML | [yamllint](https://yamllint.readthedocs.io/) |
 
-* *^^ No text linters are enabled by default.*
+* *^ No linters enabled by default.*
```

Comment:
You can use a `^` marker instead, but you should change the sentence back to what it was here. This message is supposed to tell you that for any plaintext or near-plaintext filetypes, no linters will be run by default.

We can probably enable the Vim help linters by default. The only reason the plaintext linters are off by default is because they can be very annoying when you are editing files with unknown filetypes, etc.